samples/chat/csharp-ooproc/ChatBotIsolated.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System.Net;
 using System.Text.Json;
 using System.Text.Json.Serialization;
-using Functions.Worker.Extensions.OpenAI;
+using Functions.Worker.Extensions.OpenAI.ChatBot;
```

Comment:
I'm just noticing this, but the namespace needs to start with Microsoft.Azure. Perhaps we forgot to fix this somewhere when migrating the code over from my personal repo.